src/Curl/Curl.php

```diff
@@ -96,6 +98,8 @@ public function __construct($base_url = null)
 
         $this->curl = curl_init();
         $this->id = 1;
+        $this->responseParsing = true;
```

Comment:
We should certainly allow for custom outbound return messaging/codes and 
ALSO variable handling for inbound return strings.


On 3/23/16 8:20 AM, Michael Mulligan wrote:
>
> In src/Curl/Curl.php 
> <https://github.com/php-curl-class/php-curl-class/pull/313#discussion_r57177702>:
>
> > @@ -96,6 +98,8 @@ public function __construct($base_url = null)
> >
> >          $this->curl = curl_init();
> >          $this->id = 1;
> > +        $this->responseParsing = true;
>
> I don't understand why you wouldn't just have the default to |true|. 
> By having them set in here, you may walk on an inheriting class.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub 
> <https://github.com/php-curl-class/php-curl-class/pull/313/files/76c6e92cdaa5b1b0594a2b395636a3373207517c#r57177702>
>